tensorflow/lite/micro/kernels/elementwise_test.cc

```diff
@@ -160,9 +160,9 @@ TF_LITE_MICRO_TEST(Abs) {
   tflite::testing::TestElementwiseFloat(
       tflite::BuiltinOperator_ABS,  // ABS operator
       {2, 2, 2},                    // Input shape
-      {0.01, -0.01, 10, -10},       // Input values
```

Comment:
Thanks for catching these double promotions! For consistency, can you update all the values to the form x.xf (so 10 would become 10.0f for example). Thanks.